services/app-web/src/pages/StateSubmission/ReviewSubmit/V2/ReviewSubmit/RateDetailsSummarySectionV2.test.tsx

```diff
@@ -0,0 +1,782 @@
+import { screen, waitFor, within } from '@testing-library/react'
+import {
+    mockContractPackageDraft,
+    mockContractPackageSubmitted,
+    mockMNState,
+    fetchCurrentUserMock,
+    mockValidCMSUser,
+    indexHealthPlanPackagesMockSuccess,
+} from '../../../../../testHelpers/apolloMocks'
+import { renderWithProviders } from '../../../../../testHelpers/jestHelpers'
+import { RateDetailsSummarySectionV2 as RateDetailsSummarySection } from './RateDetailsSummarySectionV2'
+import { Rate } from '../../../../../gen/gqlClient'
+import { testS3Client } from '../../../../../testHelpers/s3Helpers'
+
+describe('RateDetailsSummarySection', () => {
+    const draftContract = mockContractPackageDraft()
+    const submittedContract = mockContractPackageSubmitted()
+    const statePrograms = mockMNState().programs
+    const mockRateInfos: Rate[] = [
```

Comment:
I strongly recommend making this a function that returns this array. As is, if any test were to modify their rates, it would impact all other tests connected to these mocks.